Cubical/Categories/Instances/Sets.agda

```diff
@@ -135,3 +136,25 @@ univProp (completeSET J D) c cc =
     (λ _ → funExt (λ _ → refl))
     (λ x → isPropIsConeMor cc (limCone (completeSET J D)) x)
     (λ x hx → funExt (λ d → cone≡ λ u → funExt (λ _ → sym (funExt⁻ (hx u) d))))
+
+
+module _ {ℓ} where
+
+ open Pullback
+
+ PullbacksSET : Pullbacks (SET ℓ)
```

Comment:
This module already has a proof that SET is complete, is it feasible to get this construction as a corollary of that? Probably out of the scope of this PR but we should consider it

Reply:
No I think this construction is actually better to use than the one from the completeness proof as that one uses functions out of a finite set. But it would be good to document that in a comment here

Reply:
Added relevant comment